spec/features/trainer_spec.rb

```diff
@@ -194,7 +202,7 @@
       fill_in 'user_translation', with: 'RoR'
       click_button 'Проверить'
       expect(page).
-          to have_content 'Вы ввели не верный перевод. Повторите попытку.'
+        to have_content 'Вы ввели не верный перевод. Повторите попытку.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.